scripts/build-toolchains.sh

```diff
@@ -51,16 +52,19 @@ do
             RISCV=$(realpath $1) ;;
         --ignore-qemu )
             IGNOREQEMU="true" ;;
-	-a | --arch )
-	    shift
-	    ARCH=$1 ;;
-        riscv-tools | esp-tools)
-            TOOLCHAIN=$1 ;;
-        ec2fast )
-            EC2FASTINSTALL="true" ;;
-        * )
-            error "invalid option $1"
-            usage 1 ;;
+        --clean-after-install )
+            shift
+            CLEANAFTERINSTALL="--clean-after-install" ;;
+        -a | --arch )
+            shift
+            ARCH=$1 ;;
+              riscv-tools | esp-tools)
```

Comment:
This is mis-aligned, the `riscv-tools` and `ec2fast` sections should be indented the same as the `-a`.